docs/full-node/run-a-full-terra-node/sync.mdx

```diff
@@ -6,26 +6,134 @@ import Admonition from '@theme/Admonition';
 
+<Admonition type="caution" icon="☢️" title="Sync start times">
+
+Nodes take at least an hour to start syncing. This wait time is normal. Before troubleshooting a sync, please wait an hour for the sync to start.
+
+</Admonition>
+
+Syncing will halt at block 890,000, at which point you will need to change the version of Terrad and then resume the syncing process. 
+
+To sync your Pisco testnet node from block 890,000 to the most recent block, you will need to navigate to the `core` directory and change your Terrad version to `v2.1.0-beta.1`.
+
+```
+git checkout v2.1.0-beta.1
+make install
+```
+
+Again, make sure that you have switched to the correct version of Terrad by running the following command:
+
+```
+terrad version
+```
+
+The result of this command should be `v2.1.0`.
+
+Now, you can resume the syncing process:
+
+```
+terrad start
+```
+
+From here, you can [monitor the sync](#monitor-the-sync) and make sure to check on your node periodically to ensure optimal performance.
+
+### Phoenix-1 mainnet
+
+If you would like to sync your node using the Phoenix-1 mainnet, you will need to utilize version `v2.0.0` of Terrad up until the block height of 838,500.  To switch to this version of Terrad, change into your `core` directory and execute the following commands in your terminal:
+
+```
+git checkout v2.0.0
+make install
+```
+
+After running these commands, make sure that you have switched to the correct version of Terrad by running the command:
+
+```
+terrad version
+```
+
+The result of this command should be `v2.0.0`.
+
+After you have the correct Terrad version, you will need to download the genesis file for Phoenix-1 in the Terrad config directory:
+
+```
+cd ~/.terra/config/
+wget https://phoenix-genesis.s3.us-west-1.amazonaws.com/genesis.json
+```
+
+Finally, you can start the syncing process:
+
+```
+terrad start
+```
 
 <Admonition type="caution" icon="☢️" title="Sync start times">
 
 Nodes take at least an hour to start syncing. This wait time is normal. Before troubleshooting a sync, please wait an hour for the sync to start.
 
 </Admonition>
 
+Syncing will halt at block 838,500, at which point you will need to change the version of Terrad and then resume the syncing process. 
+
+To sync your Phoenix-1 mainnet node from block 838,500 to the most recent block, you will need to navigate to the `core` directory and change your Terrad version to `v2.1.1`.
+
+```
+git checkout v2.1.1
+make install
+```
+
+Again, make sure that you have switched to the correct version of Terrad by running the following command:
+
+```
+terrad version
+```
+
+The result of this command should be `v2.1.1`.
+
+Now, you can resume the syncing process:
+
+```
+terrad start
+```
+
+From here, you can [monitor the sync](#monitor-the-sync) and make sure to check on your node periodically to ensure optimal performance.
```

Comment:
This sentence is a run-on. You can put a comma before "and," or you can make it 2 sentences:

" From here, you can [monitor the sync](#monitor-the-sync).  Make sure to check on your node periodically to ensure optimal performance."